jsonenums.go

```diff
@@ -80,10 +82,12 @@ import (
 )
 
 var (
-	typeNames           = flag.String("type", "", "comma-separated list of type names; must be set")
-	outputPrefix        = flag.String("prefix", "", "prefix to be added to the output file")
-	outputSuffix        = flag.String("suffix", "_jsonenums", "suffix to be added to the output file")
-	exportSnakeCaseJSON = flag.Bool("snake_case_json", false, "Map camel case variable names to snake case json?")
+	typeNames              = flag.String("type", "", "comma-separated list of type names; must be set")
+	outputPrefix           = flag.String("prefix", "", "prefix to be added to the output file")
+	outputSuffix           = flag.String("suffix", "_jsonenums", "suffix to be added to the output file")
+	exportSnakeCaseJSON    = flag.Bool("snake_case_json", false, "Map camel case variable names to snake case json?")
+	serializedPrefixToDrop = flag.String("prefix_to_drop", "", "string to drop from beginning of each iota const name when converting to string")
```

Comment:
@ashultz0 There are cases where the two do not match exactly, see InventoryClassType.